src/Servers/Kestrel/test/InMemory.FunctionalTests/UpgradeTests.cs

```diff
@@ -164,23 +179,34 @@ await using (var server = new TestServer(context => Task.CompletedTask, new Test
                         "Content-Length: 1",
                         "Connection: Upgrade",
                         "",
-                        "");
+                        "A");
 
-                    await connection.ReceiveEnd(
-                        "HTTP/1.1 400 Bad Request",
-                        "Connection: close",
-                        $"Date: {server.Context.DateHeaderValue}",
-                        "Content-Length: 0",
-                        "",
-                        "");
+                    await connection.Receive("HTTP/1.1 200 OK");
+                }
+
+                using (var connection = server.CreateConnection())
+                {
+                    await connection.SendEmptyGetWithUpgrade();
+                    await connection.Receive("HTTP/1.1 200 OK");
```

Comment:
Why was this added to the test?

Reply:
Copied from another upgrade test. At one point I think there was a bug in Kestrel's upgrade logic that broke subsequent requests. These were added as regression tests.

Reply:
Maybe add a comment.

Reply:
I'll just remove it. It's covered by other tests.